pydantic/v1/_hypothesis_plugin.py

```diff
@@ -35,7 +35,7 @@
 import pydantic
 import pydantic.color
 import pydantic.types
-from pydantic.utils import lenient_issubclass
+from pydantic.v1.utils import lenient_issubclass
```

Comment:
This looks odd, but is actually fine, bc `pydantic` just uses a backport here from V1 anyways.